bin/check-redis-ping.rb

```diff
@@ -57,6 +57,12 @@ class RedisPing < Sensu::Plugin::Check::CLI
          long: '--password PASSWORD',
          description: 'Redis Password to connect with'
 
+  option :conn_failure_status,
+         long: '--conn-failure-status EXIT_STATUS',
+         description: 'Exit status for Redis connection failures',
```

Comment:
I would also maybe include that the default is crit as it is different than the others.